Makefile

```diff
@@ -17,6 +17,10 @@ docker/local/start:
 
 docker/paymentproxy/build:
 	docker build -f docker/paymentproxy/Dockerfile -t payment-proxy .
```

Comment:
There is a mismatch in names between this command and the `push` command below. Here you tag it as `payment-proxy` but below you reference it as `nitro-payment-proxy`